<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>STM32 LED Matrix Draw</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ConnectButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="StatusLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Disconnected!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="LEDTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>514</width>
      <height>258</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 87, 83);
gridline-color: rgb(136, 138, 133);</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashLine</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>16</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>16</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>16</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>16</number>
    </attribute>
   </widget>
   <widget class="QPushButton" name="DrawEraseToggleButton">
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>20</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>404</x>
      <y>20</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="CopyRightLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>330</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Made By: &lt;a href=&quot;http://RocketScientist.info&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;RocketScientist.info&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="StatusLabel_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>STM32 LED Matrix Draw - v0.9</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <action name="actionConnectButtonAction">
   <property name="text">
    <string>ConnectButtonAction</string>
   </property>
  </action>
  <action name="actionTableCellAction">
   <property name="text">
    <string>TableCellAction</string>
   </property>
  </action>
  <action name="actionDrawEraseToggleAction">
   <property name="text">
    <string>DrawEraseToggleAction</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>ConnectButton</sender>
   <signal>clicked()</signal>
   <receiver>actionConnectButtonAction</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LEDTable</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>actionTableCellAction</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DrawEraseToggleButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionDrawEraseToggleAction</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LEDTable</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>actionTableCellAction</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
